core/domain/exp_services.py

```diff
@@ -489,12 +489,22 @@ def apply_change_list(exploration_id, change_list):
             elif (
                     change.cmd ==
                     exp_domain.CMD_MIGRATE_STATES_SCHEMA_TO_LATEST_VERSION):
+                # Check that the version that the user is trying to migrate
+                # to is the latest version.
+                target_version_is_current_state_schema_version = (
+                    change.to_version ==
+                    python_utils.UNICODE(feconf.CURRENT_STATE_SCHEMA_VERSION))
+                if not target_version_is_current_state_schema_version:
+                    raise Exception(
+                        'Expected to migrate to the latest state schema '
+                        'version %s, received %s' % (
+                            feconf.CURRENT_STATE_SCHEMA_VERSION,
+                            change.to_version))
                 # Loading the exploration model from the datastore into an
```

Comment:
What is this comment describing now that the ``continue`` line is gone? Should it be moved/de-indented to be closer to the thing it's describing?

Reply:
You're right - thanks :) moved it up.